examples/buildkit3/buildkit.go

```diff
@@ -19,7 +19,7 @@ func main() {
 	var opt buildOpt
 	flag.StringVar(&opt.target, "target", "containerd", "target (standalone, containerd)")
 	flag.StringVar(&opt.containerd, "containerd", "master", "containerd version")
-	flag.StringVar(&opt.runc, "runc", "v1.0.0-rc3", "runc version")
+	flag.StringVar(&opt.runc, "runc", "v1.0.0-rc4", "runc version")
 	flag.StringVar(&opt.buildkit, "buildkit", "master", "buildkit version")
```

Comment:
and buildkit revision

Reply:
What would we use here ? a commit hash as we don't have any release yet ? :angel:

Reply:
Yes, commit hash seems fine

cc @tonistiigi

Reply:
Yes, commit hash seems fine

cc @tonistiigi